inspirehep/dojson/hep/fields/bd1xx.py

```diff
@@ -147,47 +148,29 @@ def _get_record(value):
             if x_value and x_value.isdigit():
                 return get_record_ref(x_value, 'authors')
 
-        def _get_contributor_role(value):
-            values = force_force_list(value)
-
-            contributor_roles = []
-            for value in values:
-                value = value.lower()
-                if value in current_app.config['INSPIRE_LEGACY_ROLES']['editing']:
-                    contributor_roles.append(
-                        {
-                            'schema': 'CRediT',
-                            'value': 'Writing - review & editing'
-                        }
-                    )
-                if value in current_app.config['INSPIRE_LEGACY_ROLES']['administration']:
-                    contributor_roles.append(
-                        {
-                            'schema': 'CRediT',
-                            'value': 'Project administration'
-                        }
-                    )
-            return contributor_roles
+        def _get_curated_relation(value):
+            if key.startswith('100'):
+                return value.get('y') == '1'
```

Comment:
The TWIKI is wrong. It's either 0 or 1.
https://github.com/inspirehep/inspire/blob/master/bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py#L192